web_src/js/features/repo-commit.js

```diff
@@ -62,7 +62,7 @@ export function initCommitStatuses() {
 
     createTippy(this, {
       trigger: 'click',
-      content: this.nextSibling,
+      content: this.nextElementSibling,
```

Comment:
This is the actual bug fix, `nextSibling` did include HTML text nodes,  `nextElementSibling` doesn't do this.